nav2_bt_navigator/behavior_trees/navigate_to_pose_w_replanning_and_recovery.xml

```diff
@@ -10,19 +10,21 @@
       <PipelineSequence name="NavigateWithReplanning">
```

Comment:
Ah ok, so revert this file (except the removals of the goal updated bits we talked about before). Instead, create a **new** file that contains a full BT with your only replan on invalidation with a descriptive name. 

e.g. `navigate_w_recovery_and_replanning_only_if_path_becomes_invalid.xml`

Reply:
We want to preserve the option for people that do want constant replanning

Reply:
Done!